cmake/FetchKLU.cmake

```diff
@@ -0,0 +1,29 @@
+# TODO: clean up this mess.
+include(FetchContent)
+FetchContent_Declare(
+  klu-module
+  GIT_REPOSITORY https://github.com/dpsim-simulator/SuiteSparse
+  GIT_TAG        v5.10.2
```

Comment:
this will be done seperately in SuiteSparse-repository. It has to be determined how to copy only the required files into the release, without building it.

Reply:
I have managed to release a tarball of minimal size in the SuiteSparse-repository. It is being used now